src/profile/components/modals/confirmationModal/ConfirmationModal.tsx

```diff
@@ -37,33 +37,47 @@ function ConfirmationModal({
       overlayClassName={styles.overlay}
       shouldCloseOnOverlayClick
     >
-      <div className={styles.content}>
-        <div className={styles.titleRow}>
-          <h3>{modalTitle}</h3>
-          <button
-            className={styles.closeButton}
-            type="button"
-            onClick={onClose}
-            aria-label={t('confirmationModal.close')}
-          >
-            <IconCross className={styles.icon} />
-          </button>
+      <div className={styles.wrapper}>
+        <span aria-hidden="true">
+          <IconAlertCircle />
+        </span>
+        <div className={styles.content}>
+          <div className={styles.titleRow}>
+            <h3>{modalTitle}</h3>
+            <p>{modalText}</p>
+          </div>
+          <ul>
+            {servicesArray.map((service, index) => (
+              <li key={index}>{service.title}</li>
+            ))}
+          </ul>
         </div>
-        <p>{modalText}</p>
-        <ul>
-          {servicesArray.map((service, index) => (
-            <li key={index}>{service.title}</li>
-          ))}
-        </ul>
+        <button
+          className={styles.closeButton}
+          type="button"
+          onClick={onClose}
+          aria-label={t('confirmationModal.close')}
+        >
+          <IconCross className={styles.icon} />
+        </button>
       </div>
 
       <div className={styles.actions}>
-        <Button className={styles.button} variant="outlined" onClick={onClose}>
-          {t('confirmationModal.cancel')}
-        </Button>
-        <Button className={styles.button} onClick={onConfirm}>
+        <Button
```

Comment:
Should we include some kind of loading indicator for the deletion? The HDS dialog or HDS loading button might come handy with this.